charter-draft.html

```diff
@@ -163,20 +181,27 @@ <h2 id="participation">
 					Participation
 				</h2>
 				<p>
-					Participation in PING is open to the public. Participants who do not represent a W3C Member should join as <a href="/2004/08/invexp.html" shape="rect">Invited Experts</a>. Invited Experts in this group are not granted access to Member-only information.
+					Participation in PING is open to the public. Participants who do not represent a W3C Member
+					should join as <a href="/2004/08/invexp.html" shape="rect">Invited Experts</a>. Invited Experts
+					in this group are not granted access to Member-only information.
 				</p>				
-				<p>	Anyone may subscribe to the group's public mailing list and engage in discussion. Those who intend to contribute to deliverables will be asked to join the group.
+				<p>	Anyone may subscribe to the group's public mailing list and engage in discussion. Those who
+					intend to contribute to deliverables will be asked to join the group.
 				</p>
 			</div>
 			<div class="communication">
 				<h2 id="communication">
 					Communication
 				</h2>
 				<p>
-					This group conducts its work on the public mailing list <strong><a href="https://lists.w3.org/Archives/Public/public-privacy/">public-privacy</a></strong> and in periodic phone calls, typically 1-2 times per month.
+					This group conducts its work on the public mailing list <strong><a href="https://lists.w3.org/Archives/Public/public-privacy/">public-privacy</a></strong>
+					and in periodic teleconferences, typically 1-2 times per month.  Additionally, the group meets
+					face to face at TPAC.  PING is experimenting with the use of Slack
```

Comment:
PING is experimenting with using a free Slack workspace.

Reply:
finished that sentence and added some words to reassure people about how we're using slack